examples/tutorials/tutorial_functions.py

```diff
@@ -0,0 +1,261 @@
+    backend_cfg.enable_physics = enable_physics
+
+    # sensor configurations
+    sensor_specs = []
+    for sensor_uuid, sensor_params in sensors_config_dict.items():
+        sensor_spec = habitat_sim.SensorSpec()
+        sensor_spec.uuid = sensor_uuid
+        sensor_spec.sensor_type = sensor_params["sensor_type"]
+        sensor_spec.resolution = sensor_params["resolution"]
+        sensor_spec.position = sensor_params["position"]
+        sensor_spec.orientation = sensor_params["orientation"]
+        sensor_specs.append(sensor_spec)
+
+    # agent configurations
+    agent_cfg = habitat_sim.agent.AgentConfiguration()
+    agent_cfg.sensor_specifications = sensor_specs
+
+    return habitat_sim.Configuration(backend_cfg, [agent_cfg])
+
+
+def build_multi_obs_image(
+    base_camera_img, embed_image_data_list, thumb_size, outline_frame
+):
+    """
+    Build an image containing multiple sub-images
+
+    Parameters
+    ----------
+    base_camera_img : 2d numpy array
+        Base image to insert sub images into.
+    embed_image_data_list : list of tuples
+        List of tuples where 
+            idx 0 is numpy array of image data 
+            idx 1 is boolean whether is depth data or not.
+    thumb_size : tuple of ints
+        X and Y dimensions of inset images
+    outline_frame : numpy array
+        Max X and Y dims of frame surrounding inset image
+
+    Returns
+    -------
+    result_image : 2d numpy array
+        Resultant compound image
+
+    """
+
+    result_image = base_camera_img[..., 0:3][..., ::-1]
+    x_offset = 50
+    y_offset = 50
+    y_bound = 10
+    for embed_image, embed_image_is_depth in embed_image_data_list:
+
+        if embed_image_is_depth:
+            bgr_embed_image = clip_depth_image(embed_image)
+        else:
+            bgr_embed_image = embed_image[..., 0:3][..., ::-1]
+
+        resized_bgr_embed_image = cv2.resize(
+            bgr_embed_image, thumb_size, interpolation=cv2.INTER_AREA
+        )
+
+        result_image[
+            y_offset - 1 : y_offset + outline_frame.shape[0] - 1,
+            x_offset - 1 : x_offset + outline_frame.shape[1] - 1,
+        ] = outline_frame
+        result_image[
+            y_offset : y_offset + resized_bgr_embed_image.shape[0],
+            x_offset : x_offset + resized_bgr_embed_image.shape[1],
+        ] = resized_bgr_embed_image
+        # move along y axis
+        y_offset = y_offset + y_bound + thumb_size[1]
+
+    return result_image
+
+
+def clip_depth_image(depth_image, clip_max=10.0):
+    """
+    Manually normalize depth into [0, 1] so that images are always consistent
+
+    Parameters
+    ----------
+    depth_image : 2d numpy array
+        Depth image data
+
+    Returns
+    -------
+    bgr_d_im : 2d numpy array
+        Clipped depth image data
+
+    """
+    d_im = np.clip(depth_image, 0, clip_max)
+    d_im /= clip_max
+    bgr_d_im = cv2.cvtColor((d_im * 255).astype(np.uint8), cv2.COLOR_GRAY2BGR)
```

Comment:
Are we really defaulting to BGR always? Doesn't the show image function below use RGB?

Reply:
Here again, I inherited this code from an existing tutorial and did minimal editing to content, so I cannot say why this choice was made.